drivers/bridge/bridge.go

```diff
@@ -1077,6 +1105,24 @@ func (d *driver) DeleteEndpoint(nid, eid string) error {
 		}
 	}()
 
+	// Try removal of neighbor proxy. Discard error: it is a best effort.
+	// Also make sure defer does not see this error either.
+	if n.config.NDPProxyInterface != "" && n.config.EnableIPv6 {
+		link, err := d.nlh.LinkByName(n.config.NDPProxyInterface)
+		if err == nil {
```

Comment:
I agree wit hthe best effort approach. But it would be useful to log a warning in case the link is not found (specifying this happened during neigh proxy removal)